app/routers/dogfinder.py

```diff
@@ -142,8 +154,9 @@ async def add_document(type: str = Form(...), breed: Optional[str] = Form(None),
             # Create the data object
             data_properties = create_data_properties(dogDocument)
             # Create a uuid based on the filename
-            data_properties["uuid5"] = generate_uuid5({ "filename": dogDocument.filename })
+            data_properties[uuid_field] = generate_uuid5({"filename": dogDocument.filename})
```

Comment:
No need. See comment above.

Reply:
This field is still used to put the id into the db,not just a pull thing, if I remove it I the db wont have an id to use